### PR TITLE
Fix the importClassList.pl and upgrade_admin_db.pl scripts for the move of WWSafe.pm to the pg side.

### PR DESCRIPTION
**Pull request by drgrice1**
These scripts were missed when these things were fixed before apparently.  The upgrade_admin_db.pl script is a rather critical one.

### CODE REVIEW

File: bin/importClassList.pl [hunk 1]
```diff
@@ -56,7 +63,7 @@ BEGIN
 my @replaceList =();          # Empty list
 my (@replaced, @added, @skipped);
 
-# This was copied with MINOR changes from lib/WeBWorK/ContentGenerator/Instructor/UserList2.pm 
+# This was copied with MINOR changes from lib/WeBWorK/ContentGenerator/Instructor/UserList2.pm
```
Comment (pstaabp) [inline bin/importClassList.pl:66]: `UserList2.pm` doesn't exist anymore.  If this is relevant for `UserList.pm`, else delete the comment?

Comment (drgrice1) [reply bin/importClassList.pl:66]: It is probably still relevant for `UserList.pm`.  Recall that `UserList2.pm` was moved to `UserList.pm` and the old `UserList.pm` was removed.  I will change the comment.

**Comment by pstaabp** [comment]
Can you also switch the top line of the `update_admin_db.pl` file to `#!/usr/bin/env perl`?

**Comment by pstaabp** [comment]
We should eventually change the tops of all of the scripts to this.

**Comment by drgrice1** [comment]
I changed that shebang as requested.

**Comment by pstaabp** [comment]
Not sure what I just did.  I seem to somehow merged in the previous commit.

**Comment by drgrice1** [comment]
Why is this pull request still showing up?  What did you do @pstaabp?

**Comment by drgrice1** [comment]
Did you push this on the command line?

**Pull request closed without merging**